### DISCUSSION TITLE
To much `=` in the doc about setup.cfg for src-layout projects?

### DISCUSSION THREAD

**Discussion by buhtz** [pypa/setuptools#3194] [Q&A] [answered]
My question is about this code snippet
```
[options]
package_dir=
    =src
```
Than can be found behind this link in the second source block: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#using-a-src-layout

Why is there a `=` in front of the `src` in the third line? Is this a typo or is there a reason for this?

**Comment by jwodder** [comment, marked as answer]
There's a reason.  The equivalent `setup.py` configuration is:

```
setup(
    package_dir={
        "": "src",
    },
    ...
)
```

where the empty string indicates that all of the packages in the project are under `src/` (as opposed to doing something like `package_dir={"package_name": "src"}`, which indicates that just `package_name` is under `src/`).  The leading `=` in the `setup.cfg` is then the direct translation of this syntax.

**Comment by buhtz** [comment]
Mhm... For newbie users it looks a bit wired and "wrong".
I would suggest to mention that in the docu. No need to explain the reason why there is a second `=` but just say there something like "The trailing = has to be there and is not a typo."

**Reply by abravalheri** [reply]
Hi @Codeberg-AsGithubAlternative-buhtz thank you very much for the input. Would you like to submit a PR with the documentation change?

**Reply by buhtz** [reply]
Someone who really understands whats going on should modify the docu. not me.
